### PR TITLE
Added the list of commands to readme.md and also a simple kick command based on ban 

### PR DESCRIPTION
**Pull request by ghost**
Basically the title.

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -55,6 +55,22 @@ psql trup <db/structure.sql
...
+- setfetch - Run wihout arguments to see instructions.
+- role - Use without arguments to see available roles.
+- poll - Creates a poll.
+- repo - Sends a link to the bot's repository.
+- info - Displays additional informations.
+- pfp Displays the user's profile picture in highest resolution.
+- modping - Pings online mods. (Don't abuse.)
+- desc - Sets or clears your descriptions, displays it with fetch.
+- help - Shows the list of commands and their usage.
+
```
Comment (buffet) [inline readme.md:73]: I don't think we want this, this just adds more maintenance (for example you're already missing kick, ban etc).

File: command/ban.go [hunk 2]
```diff
@@ -2,9 +2,10 @@ package command
 
 import (
 	"fmt"
-	"github.com/bwmarrin/discordgo"
 	"log"
 	"strings"
+
+	"github.com/bwmarrin/discordgo"
 )
```
Comment (buffet) [inline command/ban.go:9]: This change is inconsistent with the rest of the code base.

File: command/kick.go [hunk 3]
```diff
@@ -0,0 +1,70 @@
+package command
+
+import (
+	"fmt"
+	"log"
+	"strings"
+
+	"github.com/bwmarrin/discordgo"
+)
```
Comment (buffet) [inline command/kick.go:9]: Inconsistent with the rest of the code base.

**Review by buffet — CHANGES_REQUESTED** [review]

**Comment by buffet** [comment]
I'm beginning to think this is one of those spam PRs for the hacktber fest. How do we flag as invalid again?

**Comment by eepykate** [comment]
just add the label @buffet (maybe close too?)

**Comment by buffet** [comment]
Flagged this as invalid, looks like an attempt at getting a hacktober PR.

**Comment by eepykate** [comment]
image

wow

**Pull request closed without merging**